### DISCUSSION TITLE
trouble, KeepassXC KC gets stuck

### DISCUSSION THREAD

**Discussion by sogesehen3d** [keepassxreboot/keepassxc#11476] [Q&A]
If a URL is opened from KeepassXC, KeepassXC can no longer be used. The program can only be accessed again when the browser is closed. 
How can I stop this error, or what am I doing wrong?

KeepassXC 2.7.9

Ubuntu 24.04.1 LTS
Browser:
Firefox 132.0.2(Snap)
Vivaldi 7.0.3495.11
Chromium 130.0.6723.116(Snap)

**Comment by droidmonkey** [comment]
You might have a request dialog stuck in the background

**Reply by sogesehen3d** [reply]
Good evening droidmonkey,
that only partly looks like that. When I have called up the URL and the browser has opened, I can move the KeepassXC program window with the mouse. I just can't access the contents of the window. No other window is visible open. Terminal TOP doesn't show me anything unusual either. I have no idea what I should be looking for.

**Reply by droidmonkey** [reply]
I have never seen this behavior before. As a test, recommend downloading our appimage or switch to flatpak instead of Snap for keepassxc

**Reply by sogesehen3d** [reply]
Hello Jonathan,
I had installed KeePassXC-2.7.9-x86_64.AppImage and had the error. Now I have installed the version Snap keepassxc 2.7.8 from Jonathan White (keepassxreeboot) and Keepass is working as expected.
Can I do anything else to help locate the error? Or is this error not relevant?
In any case, thank you very much for your support.

Stefan

**Reply by droidmonkey** [reply]
Unfortunately might be something unique to how appimages are being handled on your computer. If you are gtg with snaps then just stick with that.
